File: MerchantService.Admin/app/models/branchDetail.ts
namespace Model {
    export class BranchDetail {
        Id: number;
        Name: string;
        NameSl: string;
        Code: string;
        Storename: string;
        Phone: string;
        Fax: string;
        CityId: number;
        CompanyId: number;
        spoSelection: number;
        Address: string;
        Zipcode: string;
        IsAutomaticIssueSPO: boolean;
        CreatedDateTime: Date;
        //  BranchCPOConfigurations: Array<Model.branchCPOConfigurations>;
    }
}   

File: MerchantService.DomainModel/Models/Branch/BranchDetail.cs
using MerchantService.DomainModel.Models.Company;
using MerchantService.DomainModel.Models.Global;
using System.ComponentModel.DataAnnotations.Schema;

namespace MerchantService.DomainModel.Models.Branch
{
    public class BranchDetail : MerchantServiceBase
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public string Storename { get; set; }     
        public int CompanyId { get; set; }
        public string Phone { get; set; }
        public string Fax { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public string NameSl { get; set; }      
        public string Zipcode { get; set; }
        public bool IsAutomaticIssueSPO { get; set; }
        public bool IsDelete { get; set; }
        [ForeignKey("CompanyId")]
        public virtual CompanyDetail CompanyDetail { get; set; }
    }
}
